src/main/java/com/github/mob41/blapi/RM2Device.java

```diff
@@ -111,7 +106,8 @@ public RM2Device(String host, Mac mac) throws IOException {
      */
     public boolean enterLearning() throws IOException {
         EnterLearnCmdPayload cmdPayload = new EnterLearnCmdPayload();
-        DatagramPacket packet = sendCmdPkt(10000, cmdPayload);
+        @SuppressWarnings("unused")
+		DatagramPacket packet = sendCmdPkt(10000, cmdPayload);

```

Comment:
TODO: Receive and get error_code

Reply:
I haven't done anything with learning yet. I should try and test it now that I have an RM Mini.